[2023-11-24 16:19:23,905] INFO _log() L224  [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5001 - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:19:23,905] INFO _log() L224  [33mPress CTRL+C to quit[0m - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:19:23,921] INFO _log() L224   * Restarting with stat - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:19:24,724] WARNING _log() L224   * Debugger is active! - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:19:24,725] INFO _log() L224   * Debugger PIN: 738-487-147 - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:19:51,981] INFO _log() L224  127.0.0.1 - - [24/Nov/2023 16:19:51] "OPTIONS /AvalFIMult?resgate=1&%20%20%20%20capta=2&cotistas=1000&patliq=1000000&%20%20%20%20quota=5 HTTP/1.1" 200 - - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:19:51,983] INFO _log() L224  127.0.0.1 - - [24/Nov/2023 16:19:51] "GET /AvalFIMult?resgate=1&%20%20%20%20capta=2&cotistas=1000&patliq=1000000&%20%20%20%20quota=5 HTTP/1.1" 200 - - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:46:53,509] INFO _log() L224  127.0.0.1 - - [24/Nov/2023 16:46:53] "OPTIONS /AvalFIMult?resgate=1&%20%20%20%20capta=2&cotistas=1000&patliq=1000000&%20%20%20%20quota=5 HTTP/1.1" 200 - - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:46:53,518] INFO _log() L224  127.0.0.1 - - [24/Nov/2023 16:46:53] "GET /AvalFIMult?resgate=1&%20%20%20%20capta=2&cotistas=1000&patliq=1000000&%20%20%20%20quota=5 HTTP/1.1" 200 - - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:49:49,098] INFO _log() L224  [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5001 - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:49:49,098] INFO _log() L224  [33mPress CTRL+C to quit[0m - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:49:49,100] INFO _log() L224   * Restarting with stat - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:49:49,870] WARNING _log() L224   * Debugger is active! - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:49:49,873] INFO _log() L224   * Debugger PIN: 738-487-147 - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:49:56,675] INFO _log() L224  127.0.0.1 - - [24/Nov/2023 16:49:56] "OPTIONS /AvalFIMult?resgate=1&%20%20%20%20capta=2&cotistas=1000&patliq=1000000&%20%20%20%20quota=5 HTTP/1.1" 200 - - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:49:56,675] INFO _log() L224  127.0.0.1 - - [24/Nov/2023 16:49:56] "GET /AvalFIMult?resgate=1&%20%20%20%20capta=2&cotistas=1000&patliq=1000000&%20%20%20%20quota=5 HTTP/1.1" 200 - - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:53:01,043] INFO _log() L224   * Detected change in 'C:\\venv\\.__Projetos\\PUC_EngSoft_MVP4\\back\\AvalFIMult.py', reloading - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:53:01,255] INFO _log() L224   * Restarting with stat - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:53:02,110] WARNING _log() L224   * Debugger is active! - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:53:02,113] INFO _log() L224   * Debugger PIN: 738-487-147 - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:53:08,426] INFO _log() L224  127.0.0.1 - - [24/Nov/2023 16:53:08] "OPTIONS /AvalFIMult?resgate=1&%20%20%20%20capta=2&cotistas=1000&patliq=1000000&%20%20%20%20quota=5 HTTP/1.1" 200 - - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:53:08,461] INFO _log() L224  127.0.0.1 - - [24/Nov/2023 16:53:08] "[35m[1mGET /AvalFIMult?resgate=1&%20%20%20%20capta=2&cotistas=1000&patliq=1000000&%20%20%20%20quota=5 HTTP/1.1[0m" 500 - - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:54:53,516] INFO _log() L224   * Detected change in 'C:\\venv\\.__Projetos\\PUC_EngSoft_MVP4\\back\\AvalFIMult.py', reloading - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:54:53,737] INFO _log() L224   * Restarting with stat - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:54:54,623] WARNING _log() L224   * Debugger is active! - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:54:54,628] INFO _log() L224   * Debugger PIN: 738-487-147 - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:55:02,379] INFO _log() L224  127.0.0.1 - - [24/Nov/2023 16:55:02] "OPTIONS /AvalFIMult?resgate=1&%20%20%20%20capta=2&cotistas=1000&patliq=1000000&%20%20%20%20quota=5 HTTP/1.1" 200 - - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:55:02,404] INFO _log() L224  127.0.0.1 - - [24/Nov/2023 16:55:02] "[35m[1mGET /AvalFIMult?resgate=1&%20%20%20%20capta=2&cotistas=1000&patliq=1000000&%20%20%20%20quota=5 HTTP/1.1[0m" 500 - - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:55:12,747] INFO _log() L224   * Detected change in 'C:\\venv\\.__Projetos\\PUC_EngSoft_MVP4\\back\\AvalFIMult.py', reloading - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:55:12,984] INFO _log() L224   * Restarting with stat - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:55:13,859] WARNING _log() L224   * Debugger is active! - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-24 16:55:13,864] INFO _log() L224   * Debugger PIN: 738-487-147 - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
